tests/imputers/test_BaseImputer.py

```diff
@@ -87,6 +87,15 @@ def expected_df_3():
 
         return df3
 
+    def expected_df_9():
```

Comment:
Q/N: is there a reason for this one to be in test_BaseImputer instead of test_ModeImputer?